code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -94,14 +94,12 @@
 
 	var/list/turf/synd_spawn = list()
 
-	for(var/thing in GLOB.landmarks_list)
-		var/obj/effect/landmark/A = thing
-		if(A.name == "Syndicate-Spawn")
-			synd_spawn += get_turf(A)
-			qdel(A)
-			continue
+	for(var/obj/effect/landmark/spawner/syndie/S in GLOB.landmarks_list)
+		synd_spawn += get_turf(S)
```

Comment:
This should be made into a list of XYZ at some point given that the turf ref will change if that turf ever gets changed (like putting a floor over plating). 

This is fine for now though.